index.htm

```diff
@@ -24,15 +24,74 @@
     <a href="#" class="button filled-primary-button">Sign-up</a>
   </nav>
   <div class="wrapper">
-      <div id="hero-img">
-        <div id="hero-text">
-          <h1>We Make Your Home Alive</h1>
+    <div id="hero-img">
+      <div id="hero-text">
+        <h1>We Make Your Home Alive</h1>
+      </div>
+      <div id="center-button">
+        <a href="#" class="button filled-primary-button">Shop</a>
+        <a href="#" class="button outlined-secondary-button">Blog</a>
+      </div>
+    </div>
+    <h2 id="gallery-heading">Our Loved Ones</h2>
+    <div id="gallery-container">
+      <div class="gallery-row">
+        <div class="gallery-column">
```

Comment:
![image](https://user-images.githubusercontent.com/22443504/171014643-4c3077f8-de00-4c9f-ab9c-ebc64d114b91.png)

The Scientific Name Label still has a resizing Issue